model/eventModel.js

```diff
@@ -37,4 +54,14 @@ const eventSchema = Mongoose.Schema({
     }
 });
 
+//validating one parameter based on the other one
+//pre function attaches a callback to a specified middleware function, in this case 'validate'
+eventSchema.pre('validate', function(next){
```

Comment:
Use ES6 notation.

Reply:
Arrow function didn't see the 'this' variable for some reason because the scoping of arrow and regular functions is different.

Reply:
Noted. I remember there was something similar in ReactNative.

Reply:
Yea, React Native allowed us to bind this to the method in constructor, so it was possible.